### PR TITLE
:sparkles: Export only when selection and applications are available

### PR DESCRIPTION
**Pull request by gildub**
Note. This is not about refactoring bulk ticket creation with newly available HUB API request which will be done separately.

Resolves https://github.com/konveyor/tackle2-ui/issues/914

### CODE REVIEW

**Review by ibolton336 — APPROVED** [review]

File: client/src/app/pages/migration-waves/migration-waves.tsx [hunk 1]
```diff
@@ -104,11 +104,9 @@ export const MigrationWaves: React.FC = () => {
     migrationWaveModalState !== "create" ? migrationWaveModalState : null;
   const openCreateWaveModal = () => setWaveModalState("create");
 
-  const [exportIssueModalOpen, setExportIssueModalOpen] = React.useState(false);
-
-  const [applicationsToExport, setApplicationsToExport] = React.useState<Ref[]>(
-    []
-  );
+  const [applicationsToExport, setApplicationsToExport] = React.useState<
```
Comment (ibolton336) [inline client/src/app/pages/migration-waves/migration-waves.tsx:107]: No reason to use react state here.

Comment (gildub) [reply client/src/app/pages/migration-waves/migration-waves.tsx:107]: Using only that state allowing `null` value simplifies modal handling

**Comment by gildub** [comment]
@ibolton336, the refactor is to use one state (allowing `null`) variable to handle modal.

File: client/src/app/pages/migration-waves/migration-waves.tsx (hunk 1 above)
Comment (ibolton336) [reply client/src/app/pages/migration-waves/migration-waves.tsx:107]: Ok. Can we do that without using react state? Maybe we can use a variable instead.

**Review by ibolton336 — APPROVED** [review]

**Pull request merged**